src/views/AgmExtension.vue

```diff
@@ -427,6 +427,16 @@ export default class AgmExtension extends Mixins(CommonMixin, DateMixin,
 
     const data: any = {
       [FilingTypes.AGM_EXTENSION]: {
+        // convert local properties into API/schema properties
+        year: this.data.agmYear,
+        isFirstAgm: this.data.isFirstAgm,
+        prevAgmRefDate: this.data.prevAgmDate,
+        extReqForAgmYear: this.data.isPrevExtension,
+        expireDateCurrExt: this.data.prevExpiryDate,
+        intendedAgmDate: this.data.intendedAgmDate,
+        totalApprovedExt: this.data.extensionDuration,
+        expireDateApprovedExt: this.data.agmDueDate,
+        // add in remaining local properties for future auditing
         ...this.data
```

Comment:
This is where we assign the data in the filing JSON (sent to API when filing is filed).